osquery_rules/linux/osquery_logins_non_office.yml

```diff
@@ -0,0 +1,44 @@
+AnalysisType: rule
+Filename: osquery_logins_non_office.py
+PolicyID: Osquery.Linux.LoginFromNonOffice
+DisplayName: A Login from Outside the Corporate Office
+Enabled: false
+ResourceTypes:
+  - Osquery.Differential
+Tags:
+  - Configuration Required
+  - Initial Access
+  - MITRE ATT&CK
+  - Osquery
+  - Valid Accounts
```

Comment:
For policies, we settled on the tags Availability, Configuration Required, Data Protection, Identity & Access Management, and Security Control, as well as one service based tag (e.g. AWS).

Ideally we can come up with a similar list for rules. For example, I don't like the `Valid Accounts` tag here as it seems overly specific and I don't know what other rules I would be expecting to see if I filtered by it. `Initial Access` sounds reasonable, but it may be worth taking a few minutes to step back and pick a list of tags we think will encapsulate most rules.

Outside of the scope of this PR probably, but probably good to get the brain juices flowing on this one.

Reply:
Good call, we can apple the tags you used for Policies for configuration-based rules as well (see the last commit)

For rules/behaviors, it's best to just follow MITRE ATTACK.